optimizely/decision_service.py

```diff
@@ -489,3 +489,63 @@ def get_variation_for_feature(self, project_config, feature, user_context, optio
         if rollout_variation_reasons:
             decide_reasons += rollout_variation_reasons
         return variation, decide_reasons
+
+    def validated_forced_decision(self, decision_context, user_context):
```

Comment:
can we rename it to something that's more readable and clear? like `get_validated_forced_decision`